src/kendo.data.js

```diff
@@ -1149,6 +1149,12 @@ var __meta__ = { // jshint ignore:line
             },
             isnotnull: function(a) {
                 return "(" + a + " != null)";
+            },
+            isnullorempty: function(a) {
+                return "!" + a;

```

Comment:
This operators was made only for string filter in filter cell (Kendo UI Professional). It has no sense to check number if it is "empty".
Please see another part of this request https://github.com/telerik/kendo-ui-core/issues/3488